srv/components/system/files/confstore_template.j2

```diff
@@ -47,8 +47,8 @@ server_node>{{ machine_id }}>bmc>secret=>{{ pillar['cluster'][node]['bmc']['secr
 {%- for iface in pillar['cluster'][node]['network']['mgmt']['interfaces'] %}
 server_node>{{ machine_id }}>network>management>interfaces[{{ loop.index - 1 }}]=>{{ iface }}
 {%- endfor %}
-server_node>{{ machine_id }}>network>management>gateway=>{{ salt["pillar.get"]('cluster:{0}:network:mgmt:gateway'.format(node), grains['ip4_gw']) }}
-server_node>{{ machine_id }}>network>management>netmask=>{{ pillar['cluster'][node]['network']['mgmt']['netmask'] }}
+{% set mgmt_gateway = pillar['cluster'][node]['network']['mgmt']['gateway'] %}
```

Comment:
Separate this as an independent patch.
Other changes shall merge into jenkins-mgmt-vip branch as that branch is configured for Jenkins jobs.